Osiris/Config.cpp

```diff
@@ -888,6 +889,7 @@ static void to_json(json& j, const Config::Visuals& o)
 
     WRITE("Disable post-processing", disablePostProcessing);
     WRITE("Inverse ragdoll gravity", inverseRagdollGravity);
+    WRITE("rdrawothermodels", rdrawothermodels);
```

Comment:
why bother with all of this, you cant change this anyway so just delete it from config altogether and always set drawothermodels to 100